tests/test_cases/suite.sh

```diff
@@ -27,13 +27,13 @@ echo ${deterministicinput[@]}
 echo ${nondeterministicinput[@]}
 totalarray=( "${deterministicinput[@]}" "${nondeterministicinput[@]}" )
 echo "Compiling test cases"
-x86_64-nacl-gcc-4.4.3 hello.c -o test_out/hello -std=gnu99;
+
 for var in "${totalarray[@]}"; do
     echo "Compiling test: $var"
     varnexe="${var%.*}.nexe";
-    x86_64-nacl-gcc-4.4.3 $var -o test_out/$varnexe -std=gnu99 -lpthread;
```

Comment:
why do we remove lpthread

Reply:
This must be an old change as I haven't touched the file either, I'll fix it.

Reply:
Keep lpthread in there because it's necessary for serverclient.c